internal/component/faro/receiver/server.go

```diff
@@ -85,16 +85,20 @@ func (s *server) Run(ctx context.Context) error {
 	})
 
 	mw := middleware.Instrument{
-		RouteMatcher:     r,
 		Duration:         s.metrics.requestDuration,
 		RequestBodySize:  s.metrics.rxMessageSize,
 		ResponseBodySize: s.metrics.txMessageSize,
 		InflightRequests: s.metrics.inflightRequests,
 	}
 
+	ri := middleware.RouteInjector{
```

Comment:
What is this? Is this part of the upgrade?

Reply:
The Prometheus upgrade triggered an update for dskit. And dskit had [this change](https://github.com/grafana/dskit/pull/527) which changed the way this route injector is set up.